src/session/cookies.ts

```diff
@@ -162,6 +162,7 @@ export default class Cookies {
         return cookies.filter(cookie => filterKeys.every(key => cookie[key] === filters[key]));
     }
 
+    /*eslint-disable*/
```

Comment:
You disabled linting for a part of the file.
Disable only the problematic line.